src/components/UserDetails.vue

```diff
@@ -263,6 +294,11 @@ export default {
       set(access_team_notes) {
         this.$store.commit("user/setAccessTeamNotes", access_team_notes);
       }
+    },
+    federatedIdentities: {
+      get() {
+        return this.$store.state.user.attributes.federatedIdentities;
```

Comment:
I don't know anything about this, but I'm a little surprised that `federatedIdentities` is under `attributes`. It doesn't seem to be consistent with the other data. Like, in `user.js` you set `federatedIdentities` the same way you set `email`:

```javascript
    setUser(state, user) {
        state.username = user.username
        state.firstName = user.firstName
        state.lastName = user.lastName
        state.email = user.email
        state.enabled = user.enabled
        if (user.attributes) {
            state.attributes.phone = user.attributes.phone
            state.attributes.org_details = user.attributes.org_details
            state.attributes.lockout_reason = user.attributes.lockout_reason
            state.attributes.revoked = user.attributes.revoked
            state.attributes.access_team_notes = user.attributes.access_team_notes
        }
        state.federatedIdentities = user.federatedIdentities;
    },
```

https://github.com/bcgov/moh-keycloak-user-management/blob/626bb546741d90d0e67854af9ee6732eef982d3d/src/store/modules/user.js#L38

Reply:
it's not under user attributes? it's outside of the if statement

Reply:
Right, it's not under user attributes in user.js, but in UserDetails.vue it's referenced like this:

```javascript
    federatedIdentities: {
      get() {
        return this.$store.state.user.attributes.federatedIdentities
      }
```

Compare this to how you get username:

```javascript
    username: {
      get() {
        return this.$store.state.user.username;
      },
```

I might be misunderstanding how this works, but it looks odd to me.